Week-8/exercise-8.1.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Class for creating a new node
+ */
+class Node {
+  constructor(val, next = null) {
+    this.value = val;
+    this.next = next;
+  }
+}
+
+/**
+ * Class for creating a new linked list
+ */
+class LinkedList {
+  /**
+   * Create a new linked list
+   * @param {Array} items - The items to initialize the linked list with
+   */
+  constructor(...items) {
+    this.head = null;
```

Comment:
why dont we have a member last/tail just like head?

this will help us in not recalculating the tail of the list